src/app/rundown-view/components/rundown-view/rundown-view.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnDestroy, OnInit } from '@angular/core'
+import { Rundown } from '../../../core/models/rundown'
+import { SubscriptionLike } from 'rxjs'
+import { RundownStateService } from '../../../core/services/rundown-state.service'
+import { ActivatedRoute } from '@angular/router'
+import { Logger } from '../../../core/abstractions/logger.service'
+
+@Component({
+  selector: 'sofie-rundown-view',
+  templateUrl: './rundown-view.component.html',
+  styleUrls: ['./rundown-view.component.scss'],
+})
+export class RundownViewComponent implements OnInit, OnDestroy {
+  public rundown?: Rundown
+  private rundownSubscription?: SubscriptionLike
+  private readonly logger: Logger
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly rundownStateService: RundownStateService
+  ) {}
+
+  public ngOnInit(): void {
+    const rundownId: string | null = this.route.snapshot.paramMap.get('rundownId')
+    if (!rundownId) {
+      this.logger.error("[error]: No rundownId found. Can't fetch Rundown")
+      return
+    }
+    this.rundownStateService
+      .subscribeToRundown(rundownId, rundown => {
+        this.rundown = rundown
+      })
+      .then(unsubscribeFromRundown => {
+        this.rundownSubscription = unsubscribeFromRundown
+      })
```

Comment:
Since the parameter of the lambda is a Subscription(Like), you could call the parameter for `rundownSubscription` like the class attribute.

Reply:
Changed to fit the name!